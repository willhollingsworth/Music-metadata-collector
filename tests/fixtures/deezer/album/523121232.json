{
  "id": 523121232,
  "title": "Swim",
  "upc": "4250506824938",
  "link": "https://www.deezer.com/album/523121232",
  "share": "https://www.deezer.com/album/523121232?utm_source=deezer&utm_content=album-523121232&utm_term=0_1751180348&utm_medium=web",
  "cover": "https://api.deezer.com/album/523121232/image",
  "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
  "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
  "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
  "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
  "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
  "genre_id": 106,
  "genres": {
    "data": [
      {
        "id": 106,
        "name": "Electro",
        "picture": "https://api.deezer.com/genre/106/image",
        "type": "genre"
      }
    ]
  },
  "label": "City Slang",
  "nb_tracks": 9,
  "duration": 2586,
  "fans": 629,
  "release_date": "2010-01-01",
  "record_type": "album",
  "available": true,
  "tracklist": "https://api.deezer.com/album/523121232/tracks",
  "explicit_lyrics": false,
  "explicit_content_lyrics": 0,
  "explicit_content_cover": 2,
  "contributors": [
    {
      "id": 3892,
      "name": "Caribou",
      "link": "https://www.deezer.com/artist/3892",
      "share": "https://www.deezer.com/artist/3892?utm_source=deezer&utm_content=artist-3892&utm_term=0_1751180348&utm_medium=web",
      "picture": "https://api.deezer.com/artist/3892/image",
      "picture_small": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/56x56-000000-80-0-0.jpg",
      "picture_medium": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/250x250-000000-80-0-0.jpg",
      "picture_big": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/500x500-000000-80-0-0.jpg",
      "picture_xl": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/1000x1000-000000-80-0-0.jpg",
      "radio": true,
      "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
      "type": "artist",
      "role": "Main"
    }
  ],
  "artist": {
    "id": 3892,
    "name": "Caribou",
    "picture": "https://api.deezer.com/artist/3892/image",
    "picture_small": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/56x56-000000-80-0-0.jpg",
    "picture_medium": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/250x250-000000-80-0-0.jpg",
    "picture_big": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/500x500-000000-80-0-0.jpg",
    "picture_xl": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/1000x1000-000000-80-0-0.jpg",
    "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
    "type": "artist"
  },
  "type": "album",
  "tracks": {
    "data": [
      {
        "id": 2582922122,
        "readable": true,
        "title": "Odessa",
        "title_short": "Odessa",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922122",
        "duration": 315,
        "rank": 582056,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/f/a/2/0/fa2c827a8a867828da1557868015b1a4.mp3?hdnea=exp=1751181248~acl=/api/1/1/f/a/2/0/fa2c827a8a867828da1557868015b1a4.mp3*~data=user_id=0,application_id=42~hmac=98a038b6d4a54743dae9323040e9f420852e6a48f8a74308d5ade28d77f458a9",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922132,
        "readable": true,
        "title": "Sun",
        "title_short": "Sun",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922132",
        "duration": 344,
        "rank": 431464,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/e/1/2/0/e12336d441bdbef07f0797a5c9ffe2f2.mp3?hdnea=exp=1751181248~acl=/api/1/1/e/1/2/0/e12336d441bdbef07f0797a5c9ffe2f2.mp3*~data=user_id=0,application_id=42~hmac=3c4167eb631de7f76f0ac6a5cb2bdb3f9d66fb69276fb97272315dfa4d25d16c",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922142,
        "readable": true,
        "title": "Kaili",
        "title_short": "Kaili",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922142",
        "duration": 282,
        "rank": 164289,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/b/f/6/0/bf6ab09cf1f44d6c8a6685afeeb63bf6.mp3?hdnea=exp=1751181248~acl=/api/1/1/b/f/6/0/bf6ab09cf1f44d6c8a6685afeeb63bf6.mp3*~data=user_id=0,application_id=42~hmac=011aaaa29c8fc689b2e8534a01addb30f4e1d7a7422edb496992beeb7dfba3b2",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922152,
        "readable": true,
        "title": "Found Out",
        "title_short": "Found Out",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922152",
        "duration": 198,
        "rank": 319912,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/b/2/4/0/b240e4b71f4632cc20b903d1e48fd980.mp3?hdnea=exp=1751181248~acl=/api/1/1/b/2/4/0/b240e4b71f4632cc20b903d1e48fd980.mp3*~data=user_id=0,application_id=42~hmac=867acbe000b21acef62336c4ff1b602ee4171f0adcb7073f78d4738b9e794e9a",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922162,
        "readable": true,
        "title": "Bowls",
        "title_short": "Bowls",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922162",
        "duration": 381,
        "rank": 221607,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/0/d/0/0/0d00adb95199b4bca97410c6016c53ef.mp3?hdnea=exp=1751181248~acl=/api/1/1/0/d/0/0/0d00adb95199b4bca97410c6016c53ef.mp3*~data=user_id=0,application_id=42~hmac=93489efe8a88f558f064bee01c643334869d7e273ca883d51a133f2da3fa9f34",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922172,
        "readable": true,
        "title": "Leave House",
        "title_short": "Leave House",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922172",
        "duration": 312,
        "rank": 308636,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/c/e/3/0/ce38dd7a060071f61c57ca5e49fa537a.mp3?hdnea=exp=1751181248~acl=/api/1/1/c/e/3/0/ce38dd7a060071f61c57ca5e49fa537a.mp3*~data=user_id=0,application_id=42~hmac=32e75b80a5bbf00cf610003bb546b8272148e981ee5f5b4a2a2d6e1c43dc4833",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922182,
        "readable": true,
        "title": "Hannibal",
        "title_short": "Hannibal",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922182",
        "duration": 375,
        "rank": 228776,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/f/e/1/0/fe194978b7862b5a1e20a55dd5051aed.mp3?hdnea=exp=1751181248~acl=/api/1/1/f/e/1/0/fe194978b7862b5a1e20a55dd5051aed.mp3*~data=user_id=0,application_id=42~hmac=080d65eaafe240c8e8801dd2a8986f8c47c0f183eb2a9764e05d159bf19d9f20",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922192,
        "readable": true,
        "title": "Lalibela",
        "title_short": "Lalibela",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922192",
        "duration": 145,
        "rank": 252042,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/9/3/9/0/939e30b50902fbe7e5cba90b51626f83.mp3?hdnea=exp=1751181248~acl=/api/1/1/9/3/9/0/939e30b50902fbe7e5cba90b51626f83.mp3*~data=user_id=0,application_id=42~hmac=a83a158a852abac2aa64e407f7767125c7a0619f394bc01b3793f16e07c9abab",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      },
      {
        "id": 2582922202,
        "readable": true,
        "title": "Jamelia",
        "title_short": "Jamelia",
        "title_version": "",
        "link": "https://www.deezer.com/track/2582922202",
        "duration": 234,
        "rank": 175371,
        "explicit_lyrics": false,
        "explicit_content_lyrics": 0,
        "explicit_content_cover": 2,
        "preview": "https://cdnt-preview.dzcdn.net/api/1/1/f/5/f/0/f5f07813981996fa92af1d507c0a4d62.mp3?hdnea=exp=1751181248~acl=/api/1/1/f/5/f/0/f5f07813981996fa92af1d507c0a4d62.mp3*~data=user_id=0,application_id=42~hmac=63221b0a5a6e408110a073dcfad25193dc8734ab413ef4b47d38c8b8e5be1006",
        "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
        "artist": {
          "id": 3892,
          "name": "Caribou",
          "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
          "type": "artist"
        },
        "album": {
          "id": 523121232,
          "title": "Swim",
          "cover": "https://api.deezer.com/album/523121232/image",
          "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
          "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
          "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
          "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
          "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
          "tracklist": "https://api.deezer.com/album/523121232/tracks",
          "type": "album"
        },
        "type": "track"
      }
    ]
  }
}