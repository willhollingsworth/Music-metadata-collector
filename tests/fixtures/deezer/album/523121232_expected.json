{
  "album_name": "Swim",
  "album_id": 523121232,
  "artist_name": "Caribou",
  "artist_id": 3892,
  "track_count": 9,
  "fans": 629,
  "release_date": "2010-01-01",
  "link": "https://www.deezer.com/album/523121232",
  "genres": [
    "Electro"
  ]
}